### DISCUSSION TITLE
how to make separate Claim Blocks Accrued Per Hour

### DISCUSSION THREAD

**Discussion by Anan4sek** [GriefPrevention/GriefPrevention#2155] [Q&A]
how to make separate Claim Blocks Accrued Per Hour settings for different groups, e.g. vip, player, etc., when I added it in the config, it was removed after a restart

**Comment by QarthO** [comment]
You'll need an addon for this, if you check the discussions tab, i believe there already exists an addon for this

See this: https://github.com/TechFortress/GriefPrevention/discussions/2075
